src/main.rs

```diff
@@ -61,7 +59,23 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let host = IpAddr::from_str(&host_raw)?;
     let port = env::var("PORT").unwrap_or_else(|_| "80".into()).parse()?;
 
-    let client: Client<_, Body> = Client::builder().build(HttpsConnector::with_webpki_roots());
+    let https_connector = {
+        let mut http_connector = new_trust_dns_http_connector();
+        http_connector.enforce_http(false);
+
+        let builder = HttpsConnectorBuilder::new()
+            .with_webpki_roots()
+            .https_only()
+            .enable_http1();
+
+        if env::var("DISABLE_HTTP2").is_ok() {
```

Comment:
Should also check if it is set to `1` to line up with the documentation in the README

Reply:
I don't really think it matters, this is a flag-style env var and doesn't require a value. `DISABLE_HTTP2=true` or `DISABLE_HTTP2=yes` should work just as fine

Reply:
`DISABLE_HTTP2=no` or `DISABLE_HTTP2=0` also both disables http2, Either the documentations should be rewritten to make it clear it is as long as it is set or it should be a 0-1 flag

Reply:
changed the wording to make it clear that any value for `DISABLE_HTTP2` will toggle this